src/labs_examples/labs/arrays/MultiDimArrays.java

```diff
@@ -0,0 +1,15 @@
+package labs_examples.labs.arrays;
+
+public class MultiDimArrays {
+    public static void main(String[] args) {
+        int[][] twoArrays = new int[4][7];
+        for (int i = 0; i < twoArrays.length; i++) {
+            for (int j = 0; j < twoArrays[i].length; j++) {
+                //why multiply them
```

Comment:
I believe just to show that array folds different values. :)

Reply:
Im still a little confuse it just checks if it matches? thank you

Reply:
Based on the code it:
- assigns product of I and J to corresponding cell
- prints the cell content

So I assume it is just a demonstration of the multidimensional arrays